prow/cluster/jobs/istio-releases/daily-release/istio-releases.daily-release.release-1.1.yaml

```diff
@@ -0,0 +1,158 @@
+job_template: &job_template
+  branches:
+  - "^release-1.1$"
```

Comment:
Use `*branch_spec` here?

Reply:
What does that mean here?

Reply:
This value is available as a YAML anchor defined here: https://github.com/istio/test-infra/pull/1009/files/7788653250677e1196f2e8a019c592cc3014f909#diff-d95c614ae3e0f650eed2e9c06b138d1eR46

Reply:
Got it. Updated a few other files too for consistency.